test/physics/em/PhotoelectricInteractor.test.cc

```diff
@@ -0,0 +1,237 @@
+        constexpr auto zero   = celeritas::zero_quantity();
+        constexpr auto stable = ParticleDef::stable_decay_constant();
+
+        // Set up shared particle data
+        Base::set_particle_params(
+            {{"electron",
+              pdg::electron(),
+              MevMass{0.5109989461},
+              ElementaryCharge{-1},
+              stable},
+             {"gamma", pdg::gamma(), zero, zero, stable}});
+
+        const auto& params    = this->particle_params();
+        pointers_.electron_id = params.find(pdg::electron());
+        pointers_.gamma_id    = params.find(pdg::gamma());
+        pointers_.inv_electron_mass
+            = 1 / (params.get(pointers_.electron_id).mass.value());
+
+        // Set Livermore photoelectric data
+        LivermoreParams::Input li;
+        std::string data_path = this->test_data_path("physics/em", "");
+        LivermoreParamsReader read_element_data(data_path.c_str());
+        li.elements.push_back(read_element_data(19));
+        set_livermore_params(li);
+
+        // Set default particle to incident 10 MeV photon
+        this->set_inc_particle(pdg::gamma(), MevEnergy{10});
+        this->set_inc_direction({0, 0, 1});
+
+        // Set up shared material data
+        MaterialParams::Input mi;
+        mi.elements  = {{19, AmuMass{39.0983}, "K"}};
+        mi.materials = {{1e-5 * na_avogadro,
+                         293.,
+                         MatterState::solid,
+                         {{ElementDefId{0}, 1.0}},
+                         "K"}};
+
+        // Set default material to potassium
+        this->set_material_params(mi);
+        this->set_material("K");
+    }
+
+    void sanity_check(const Interaction& interaction) const
+    {
+        ASSERT_TRUE(interaction);
+
+        // Check change to parent track
+        EXPECT_GT(this->particle_track().energy().value(),
+                  interaction.energy.value());
+        EXPECT_EQ(celeritas::Action::absorbed, interaction.action);
+
+        // Check secondaries
+        ASSERT_GT(2, interaction.secondaries.size());
+        if (interaction.secondaries.size() == 1)
+        {
+            const auto& electron = interaction.secondaries.front();
+            EXPECT_TRUE(electron);
+            EXPECT_EQ(pointers_.electron_id, electron.def_id);
+            EXPECT_GT(this->particle_track().energy().value(),
+                      electron.energy.value());
+            EXPECT_LT(0, electron.energy.value());
+            EXPECT_SOFT_EQ(1.0, celeritas::norm(electron.direction));
+        }
+
+        // Check conservation between primary and secondaries
+        // TODO: atomic relaxation has not been implemented. If the binding
+        // energy of the sampled shell is larger than the incident photon
+        // energy, or there is an energy balance following atomic deexcitation,
+        // the energy will be deposited locally.
+        // this->check_conservation(interaction);
```

Comment:
I forgot if we discussed this already, but perhaps now's the time to add a `local_energy_deposition` member to the `Interaction`? @whokion @stognini @pcanal thoughts?